### PR TITLE
Update to the latest dev-support defaults

### PR DESCRIPTION
**Pull request by elia**

### CODE REVIEW

File: solidus_product_assembly.gemspec [hunk 1]
```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.executables = files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'solidus_core', ['>= 2.0.0', '< 4']
+  spec.add_dependency 'solidus_core', ['>= 2.0.0', '< 5']
```
Comment (elia) [inline solidus_product_assembly.gemspec:32]: Thoughts on bumping to `>= 3`?

**Review by elia — COMMENTED** [review]
✅ @kennyadsl can you approve it for me?

**Review by rainerdema — APPROVED** [review]
👏

File: Rakefile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+ENV['SKIP_SOLIDUS_BOLT'] = 'true'
```
Comment (waiting-for-dev) [inline Rakefile:3]: I think there's no need for that, as it'll be already skipped if there's no Gemfile, as in the case of the dummy app: https://github.com/solidusio/solidus_frontend/blob/aaf0f7db868c69866036dbdd09484783e70907ab/lib/generators/solidus_frontend/install/install_generator.rb#L46

**Review by waiting-for-dev — APPROVED** [review]

File: solidus_product_assembly.gemspec (hunk 1 above)
Comment (kennyadsl) [reply solidus_product_assembly.gemspec]: Done!

**Pull request merged**